manifests/puppetserver.pp

```diff
@@ -18,7 +18,7 @@
     override_metrics_command => $override_metrics_command,
   }
 
-  if versioncmp($facts['pe_server_version'], '2018.1.0') < 0 {
+  if ($facts['pe_server_version'] =~ NotUndef) and (versioncmp($facts['pe_server_version'], '2018.1.0') < 0) {
```

Comment:
Do we want to add a parameter that would allow users to enable these if they are on a new enough version of puppetserver?

Reply:
This will be enabled for open source Puppet Server 5.x or newer. I promoted these metrics to the `/status` API in that version. The `pe_server_version` check here is testing for older versions of PE and uses the `/metrics` API to retrieve the data there.

Most Puppet Server metrics were "PE only" prior to 5.x, so there isn't much to gain by backporting this specific check, but we should think about how to do version-specific metrics in a way that supports both PE and Open Source.